.circleci/config.yml

```diff
@@ -47,6 +47,7 @@ jobs:
           command: curl localhost:${PORT}/_status/check -I
       - run:
           name: Snapshot screenshots
+          no_output_timeout: 30m
```

Comment:
Does this help?

As far as I understand the `no_output_timeout` is to say how long a command may run without output. But I don't think Percy runs 30m without any output.  https://circleci.com/docs/configuration-reference/#run

The failures we've seen recently were not caused by command running too long I think.